seqr/management/tests/load_rna_seq_tests.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from seqr.models import Sample, RnaSeqOutlier
+from seqr.views.utils.orm_to_json_utils import get_json_for_rna_seq_outliers
+
+RNA_FILE_ID = 'tmp_-_2021-03-01T00:00:00_-_test_data_manager_-_new_muscle_samples.tsv.gz'
+EXISTING_SAMPLE_GUID = 'S000150_na19675_d2'
+
+class LoadRnaSeqTest(TestCase):
+    fixtures = ['users', '1kg_project', 'reference_data']
+
+    @mock.patch('seqr.views.apis.data_manager_api.ANALYST_PROJECT_CATEGORY', 'analyst-projects')
+    @mock.patch('seqr.management.commands.load_rna_seq.logger.info')
+    @mock.patch('seqr.management.commands.load_rna_seq.open')
+    @mock.patch('seqr.views.apis.data_manager_api.gzip.open')
+    def test_command(self, mock_gzip_open, mock_open, mock_logger):
```

Comment:
It could be cleaner to mock 'seqr.views.apis.data_manager_api.load_rna_seq'.

Reply:
I would rather actually test that the function is being called properly. If you mock the entire function and the function signature changes your tests pass because you mocked the functions but the real world behavior fails because its not getting the expected arguments